example/package.json

```diff
@@ -23,13 +23,15 @@
   "alias": {
     "react": "../node_modules/react",
     "react-dom": "../node_modules/react-dom/profiling",
-    "scheduler/tracing": "../node_modules/scheduler/tracing-profiling"
+    "scheduler/tracing": "../node_modules/scheduler/tracing-profiling",
+    "@rainbow-me/rainbow-button": "../."
```

Comment:
smart, is taking the `dist` folder?

Reply:
I think so. It occurred to me that I should double check development watching flow is still working though. As far as the built website is concerned though, it has the rainbow button code in the bundle.

Reply:
👌